### ISSUE TITLE
integrate audio into the app

### ISSUE THREAD

**Issue by alee** [virtualcommons/port-of-mars#647]
- clear sound / notification when the game starts
- sound effects for different game actions (chat, investment, purchase, discard), events
- overall game music
- allow user to enable/disable sound fx / music?

**Comment by chrstngyn** [comment]
https://freesound.org/

**Comment by chrstngyn** [comment]
https://github.com/vueuse/sound

**Comment by chrstngyn** [comment]
I don't think we need to use @vueuse/sound. We can create an audio object instead https://developer.mozilla.org/en-US/docs/Web/API/HTMLAudioElement/Audio

Demonstrated in e9eb62cf8e403068f09cc37e87162536aed44a26

**Comment by chrstngyn** [comment]
fs node module (provides functionality to access and interact with file system) can be used on the server not the client. we may have to generate sound objects on the serverside instead

**Comment by alee** [comment]
JS audio libraries to look at:

https://howlerjs.com/ 

https://alemangui.github.io/pizzicato/

https://createjs.com/soundjs

**Comment by cpritcha** [comment]
We tried out howlerjs which works well.

We need to think about sound handling.

- Some sounds are generated on the client only like if we have a click noises on the investment time block add and subtract buttons.
- Other sounds are generated in response events like a change in phase, the beginning of a new game

Reacting to mutations in the Vuex store will result in playing sounds when we reload so I lean towards generating special events from the server that the client reacts to.
